bedrock/mozorg/templates/mozorg/about/governance/policies/social-content-policies.html

```diff
@@ -42,8 +42,7 @@ <h3>How to submit a copyright notice</h3>
   <p>
     Mozilla will respond to takedown requests that comply with the Digital Millennium Copyright Act (DMCA).
     If you believe that content shared on the Mozilla.Social server infringes a copyright owned by you, or
-    by someone that you legally represent, you may submit a takedown notice, using the procedures outlined
-    on <a href="{{ url('legal.report-infringement') }}">Mozilla’s copyright page</a>.
+    by someone that you legally represent, you may submit a takedown notice using this <a href="https://mozilla.social/en-US/about/legal/report-infringement-form" target="_blank" rel="external noopener">form</a>. For more information about our copyright policy and review process, see <a href="{{ url('legal.report-infringement') }}">Mozilla’s copyright page</a>.
```

Comment:
This URL redirects me to `https://reports.mozilla.social/infringement-form`; should we link directly there?

Reply:
Ah, yeah. Let me ask